web/src/main/kotlin/no/nav/su/se/bakover/web/routes/person/PersonRoutes.kt

```diff
@@ -28,33 +30,39 @@ internal fun Route.personRoutes(
     personService: PersonService,
     clock: Clock
 ) {
-    get("$personPath/{fnr}") {
-        call.lesFnr("fnr").fold(
-            ifLeft = { call.svar(BadRequest.message(it)) },
-            ifRight = { fnr ->
-                call.svar(
-                    personService.hentPerson(fnr).fold(
-                        {
-                            when (it) {
-                                FantIkkePerson -> Resultat.message(NotFound, "Fant ikke person")
-                                IkkeTilgangTilPerson -> Resultat.message(
-                                    HttpStatusCode.Forbidden,
-                                    "Ikke tilgang til å se person"
-                                )
-                                Ukjent -> Resultat.message(
-                                    HttpStatusCode.InternalServerError,
-                                    "Feil ved oppslag på person"
-                                )
+    post("$personPath/søk") {
+        data class Body(
+            val fnr: String
+        )
+
+        call.withBody<Body> { body ->
+            Either.catch { Fnr(body.fnr) }.fold(
+                ifLeft = { call.svar(BadRequest.message("${body.fnr} er ikke et gyldig fødselsnummer")) },
```

Comment:
Generelt ønske: Bruke .jsonMessage istedet for .message slik at framover på sikt kan mappe de til en feilkode

Reply:
Ahja, jeg tok bare det som var der fra før (inne i `lesFnr` i Extensions).